app/components/security_key_image_component.scss

```diff
@@ -0,0 +1,99 @@
+@use 'uswds-core' as *;
```

Comment:
Optional: I don't think we're using any design system here, so we could remove this. In theory it could help improve compilation time, but it might not make a huge difference after #9954 .

Reply:
Will remove! I thought it was needed when I was using the `'mobile'` token break earlier

```suggestion
```